docs/guides/cli-completions.md

```diff
@@ -0,0 +1,39 @@
+# CLI Completions
+`lockbook` ships with a `_lockbook` tab completions file.
+In general, your shell scans `$FPATH` for completions.
+
+## Debugging CLI Completions
+If `lockbook` completions are not working automatically for you, check out [homebrew's](https://docs.brew.sh/Shell-Completion) guide.
+
+## Manual Creation
+If building from source or your package manager doesn't support completions, `lockbook` supports manual creation
+### bash
+#### Auto-loaded
+```
+lockbook completions bash >> ~/.bash_completion
+```
+#### Lazy-loaded
+```
+lockbook completions bash > ${XDG_DATA_HOME:-~/.local/share}/bash-completion/completions/lockbook
+```
+### fish
+```
+lockbook completions fish > ~/.config/fish/completions/lockbook.fish
+```
+### zsh
+- `oh-my-zsh` note: ensure you modify your `$FPATH` before `source $ZSH/oh-my-zsh.sh` because it will call `compinit` for you.
+```
+lockbook completions zsh > /usr/local/share/zsh/site-functions/_lockbook
+```
+
+## Build System
```

Comment:
this is just what happens on `aur`, I'm not sure that code snippet is the most relevant. Additionally this link only applies to what happens on linux, there's a corresponding link for what happens on macOS (brew).